kong/plugins/http-log/schema.lua

```diff
@@ -8,15 +11,14 @@ return {
     { config = {
         type = "record",
         fields = {
-          -- NOTE: any field added here must be also included in the handler's get_queue_id method
           { http_endpoint = typedefs.url({ required = true, encrypted = true, referenceable = true }) }, -- encrypted = true is a Kong-Enterprise exclusive feature, does nothing in Kong CE
           { method = { type = "string", default = "POST", one_of = { "POST", "PUT", "PATCH" }, }, },
           { content_type = { type = "string", default = "application/json", one_of = { "application/json" }, }, },
           { timeout = { type = "number", default = 10000 }, },
           { keepalive = { type = "number", default = 60000 }, },
-          { retry_count = { type = "integer", default = 10 }, },
-          { queue_size = { type = "integer", default = 1 }, },
-          { flush_timeout = { type = "number", default = 2 }, },
+          { retry_count = { type = "integer", default = 10 }, },  -- deprecated, use queue.max_retry_time
+          { queue_size = { type = "integer", default = 1 }, }, -- deprecated, use queue.batch_max_size
+          { flush_timeout = { type = "number", default = 2 }, }, -- deprecated, use queue.max_delay
```

Comment:
Is this a deprecation or a breaking change?
Deprecation implies that the setting is still honored and won't be honored in a future version.

Reply:
This is now handled with an appropriate deprecation message in each plugin's schema.